code/modules/mob/living/carbon/human/human.dm

```diff
@@ -240,22 +240,24 @@
 				limbs_amount = 4
 
 		if(EXPLODE_HEAVY)
+			var/stuntime = 5 SECONDS - (bomb_armor / 20) SECONDS //Between no stun and 5 seconds of stun depending on bomb armor
```

Comment:
```suggestion
			var/stuntime = 5 SECONDS - (bomb_armor / 2) //Between no stun and 5 seconds of stun depending on bomb armor
```
you could do this instead, is 1 less operation. and I think it might be slightly more readable? `(bomb_armor / 20) SECONDS` feels rather abstract

Reply:
I am so fucking stupid